src/tooltip/test/tooltip.spec.js

```diff
@@ -249,10 +249,11 @@ describe('tooltip', function() {
       scope.$digest();
     }));
 
-    it('should open after timeout', inject(function ($timeout) {
+    it('should open after timeout and set the close delay to the default', inject(function ($timeout) {
 
       elm.trigger('mouseenter');
       expect(tooltipScope.isOpen).toBe(false);
```

Comment:
By direct testing I mean the `display` property of the element. Wouldn't that be more reliable. It is true that this is how it is tested there, but maybe adding such a display test would help to guard against future bugs. Anyway, was just a suggestion.